src/modules/Dashboard/features/Applet/Overview/QuickStats/QuickStats.styles.ts

```diff
@@ -0,0 +1,24 @@
+import styled from '@emotion/styled';
+import { Box } from '@mui/material';
+
+import { theme, variables } from 'shared/styles';
+
+export const StyledRoot = styled(Box)({
+  borderRadius: variables.borderRadius.lg2,
+  boxShadow: '0px 0px 16px 0px #00000014',
```

Comment:
Good call out — I did a search and could find the equivalent styles being used elsewhere, so I added it as a one-off as shown. Since design has a specific token for this though, I went ahead and added one as well.